<?xml version="1.0"?>
<launch>

    <node pkg="server_update" type="server_update_node.py" name="server_update_node" output="screen">
        <param name="server" value="http://www.lassnig.cc/home" />
      	<param name="topic_log" value="/log" />
      	<param name="topic_events" value="/events" />
      	<param name="topic_stats" value="/stats" />
      	<param name="ros_sleep_time" value="5.0" />
    </node>

</launch>
